pandas/tests/reshape/test_qcut.py

```diff
@@ -301,3 +301,17 @@ def test_qcut_nullable_integer(q, any_nullable_int_dtype):
     expected = qcut(arr.astype(float), q)
 
     tm.assert_categorical_equal(result, expected)
+
+
+@pytest.mark.parametrize(
+    "Data_type,Data_type_string",
+    [(pd.Float64Dtype(), "Float64Dtype"), (pd.Int64Dtype(), "Int64Dtype")],
+)
+def test_qcut_numeric_dtype(Data_type, Data_type_string):
+    series = pd.Series([1.0, 2.0, 3.0, 4.0], dtype=Data_type)
+
+    try:
+        pd.qcut(series, 2)
+    except:
+        Fail_string = Data_type_string + " is not supported"
+        pytest.fail(msg=Fail_string)
```

Comment:
What are you trying to test here? If you're to test that something works, this try/except structure is not needed, if it actually raises the test will just fail as desired.

It might be helpful to try to follow examples from above - construct result from using `qcut`, then compare to an `expected` result that you hardcode.

I'd recommend just folding into the test above - you could just parameterize the test above by `any_nullable_numeric_dtype` type instead of `any_nullable_int_dtype` rather than adding a new test.

Reply:
Got it. thanks